examples/manageddisk_lustre/config.json

```diff
@@ -0,0 +1,257 @@
+{
+  "location": "westeurope",
+  "resource_group": "variables.resource_group",
+  "install_from": "headnode",
+  "admin_user": "hpcadmin",
+  "vnet": {
+    "name": "hpcvnet",
+    "address_prefix": "10.2.0.0/20",
+    "subnets": {
+      "compute": "10.2.0.0/22",
+      "storage": "10.2.4.0/24"
+    }
+  },
+  "variables": {
+    "resource_group": "azhpc-cluster",
+    "lustre_disk_type": "data_disk",
+    "image": "OpenLogic:CentOS:7.6:latest",
+    "hpcimage": "OpenLogic:CentOS-HPC:7.6:latest",
+    "storage_account": "xxxxxxxx",
+    "storage_key": "/xxxxxxxxxxxxx",
+    "storage_container": "xxxxxxxxx",
+    "log_analytics_lfs_name": "xxxxxxx",
+    "log_analytics_workspace": "xxxxxxxx",
+    "log_analytics_key": "xxxxxxx",
+    "location": "westeurope"
+  },
+  "resources": {
+    "headnode": {
+      "type": "vm",
+      "vm_type": "Standard_HB60rs",
```

Comment:
Please don't use HB for Headnode for cost savings

Reply:
But this is what was done for all the Lustre versions for azhpc, not just here. I need a clear determination for the compute nodes.

Reply:
Please start by changing this one, we will change the others too.

Reply:
I thought the decision was to use HB nodes.

Reply:
No because this is the most expensive choice.

Reply:
Please discuss with Paul. Changes can be made directly to the config.json as determined.